openy_map.install

```diff
@@ -148,9 +148,19 @@ function openy_map_update_8008() {
     ->save(TRUE);
 }
 
- /**
+/**
  * Update path to icons.
  */
 function openy_map_update_8009() {
   _openy_map_absolutize_config_icon_urls();
 }
+
+/**
+ * Adds default language condition to the view location.
+ */
+function openy_map_update_8010() {
+  $config_path = \Drupal::service('extension.path.resolver')->getPath('module', 'openy_map') . '/config/install';
+  $source = new FileStorage($config_path);
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('views.view.locations', $source->read('views.view.locations'));
```

Comment:
please, use config updater @andrebonon https://github.com/open-y-subprojects/openy_map/pull/74/files#diff-f5f5f7f0a35898108deac0ddd5770f86a2910ef5c8f0be5ac7fb7b499d5d6804R118

config write not always run proper subscriber, and not all of the,m